### DISCUSSION TITLE
Ci sumiu

### DISCUSSION THREAD

**Discussion by ItaloCobains** [DesignLiquido/delegua#158] [P&R] [answered]
### O ci sumiu.

foi de proposito ?

![image](https://user-images.githubusercontent.com/83548040/200095704-95a5f24b-19d7-43d3-ae88-2374a8b3f9d6.png)

**Comment by samuelrvg** [comment, marked as answer]
Está falando das `Actions` ?, se sim, esta ao lado da aba de `Discussions` ?
